### PR TITLE
lose code shorty

### LINKED ISSUE #4038: increase varchar size for code_text and code_text_short in codes tables
### Describe the problem

Facility bills for ABA therapy and their CPT codes run long compared to space given. Their code_text_short averages 48 characters and their code_text bumps into 441 characters.

code_text_short examples:
`Adaptive Behavior Treatment with Protocol Modification`
`Behavior Identification Supporting Assessment`
`Multiple-family group adaptive behavior treatment guidance`
etc...

code_text example:
`administered by a physician or other qualified healthcare professional, each 15 minutes of the physician’s or other qualified healthcare professional’s time face-to-face with patient and/or guardian(s)/caregiver(s) administering assessments and discussing findings and recommendations, and non face-to-face analyzing past data, scoring/interpreting the assessment, and preparing the report/treatment plan.`

### Describe the solution you'd like

Could the varchar field size for `code_text_short` in `codes` table be increased from 24 to 255 as the default? It shouldn't affect the table size ( I think).
Could the varchar field size for `code_text` in `codes` table be increased from 255 to 1024 as the default? It would add an extra byte of storage space to each record stored in that field ( I think).

Following the steps in this wiki for inputting the data: [https://www.open-emr.org/wiki/index.php/Code_Types](https://www.open-emr.org/wiki/index.php/Code_Types)

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #4038

### CODE REVIEW

File: sql/5_0_2-to-6_0_0_upgrade.sql [hunk 1]
```diff
@@ -461,8 +461,8 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`
 ALTER TABLE `form_eye_mag_prefs` MODIFY `ordering` smallint(6) DEFAULT NULL;
 #EndIf
 
-#IfNotColumnType codes code_text_short varchar(255)
-ALTER TABLE `codes` MODIFY `code_text_short` varchar(255) NOT NULL default '';
+#IfNotColumnType codes code_text_short text
+ALTER TABLE `codes` MODIFY `code_text_short` text NOT NULL default '';
```
Comment (bradymiller) [inline sql/5_0_2-to-6_0_0_upgrade.sql:465]: the nice thing about text is it just gets to be by itself:
```
ALTER TABLE `codes` MODIFY `code_text_short` text;
```
(would do the same for the other places that are changed to text in this pr)

**Comment by bradymiller** [comment]
do you think we should do the same thing to the codes_history table (I am not really sure that that table even does)?

**Comment by stephenwaite** [comment]
good pt, better since it's enabled by default, looks like just used from the codes administration

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
2DV

**Pull request merged**